pkg/cvo/metrics.go

```diff
@@ -346,8 +356,27 @@ func (m *operatorMetrics) Describe(ch chan<- *prometheus.Desc) {
 	ch <- m.clusterOperatorConditionTransitions.WithLabelValues("", "").Desc()
 	ch <- m.clusterInstaller.WithLabelValues("", "", "").Desc()
 	ch <- m.clusterVersionOperatorUpdateRetrievalTimestampSeconds.WithLabelValues("").Desc()
+	ch <- m.clusterVersionConditionalUpdatesRecommendedConditions.WithLabelValues("", "", "").Desc()
 }
 
+func (m *operatorMetrics) collectConditionalUpdates(ch chan<- prometheus.Metric, updates []configv1.ConditionalUpdate) {
+	for _, update := range updates {
+		for _, condition := range update.Conditions {
+			if condition.Type != ConditionalUpdateConditionTypeRecommended {
+				continue
+			}
+
+			g := m.clusterVersionConditionalUpdatesRecommendedConditions
+			gauge := g.WithLabelValues(update.Release.Version, strings.ToLower(string(condition.Status)), condition.Reason)
```

Comment:
why the cast `ToLower`?

Reply:
Personal preference for lowercase boolean terms. I have removed it.